backend/src/test/java/quarano/department/web/EnrollmentWebIntegrationTests.java

```diff
@@ -1,10 +1,13 @@
 package quarano.department.web;
 
-import static org.assertj.core.api.Assertions.*;
-import static org.springframework.restdocs.hypermedia.HypermediaDocumentation.*;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
-import static quarano.department.web.TrackedCaseLinkRelations.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.restdocs.hypermedia.HypermediaDocumentation.linkWithRel;
+import static org.springframework.restdocs.hypermedia.HypermediaDocumentation.relaxedLinks;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static quarano.department.web.TrackedCaseLinkRelations.CONFIRM;
+import static quarano.department.web.TrackedCaseLinkRelations.ENROLLMENT;
```

Comment:
Please use the usual settings for imports: https://quarano.de/internal-test/api/docs/index.html#_import_order